src/lib.rs

```diff
@@ -0,0 +1,18 @@
+#![allow(non_snake_case)]
+#![warn(unused_crate_dependencies)]
+
+/// Phoenix's Core Errors
+pub mod error;
+/// Macros
+pub mod macros;
+/// Transparent and Obfuscated Notes
+pub mod note;
+
+pub use error::Error;
+pub use note::{Note, NoteType};
+
+use dusk_plonk::bls12_381::Scalar as BlsScalar;
+use dusk_plonk::jubjub::{
+    AffinePoint as JubJubAffine, ExtendedPoint as JubJubExtended,
+    Fr as JubJubScalar,
+};
```

Comment:
Too verbose.

Use:
```rust
use dusk_plonk::prelude::*;
```

Reply:
Solving this because we discussed about when to use `prelude` and when not (basically for `dusk_plonk` prelude is necessary only when we're going to use the composer / plonk, not when we want to access only to the re-exported crates).

For future reference, we might change this approach in the future (but is not something we're going to do atm).